### PR TITLE
2058662: Fix inaccurate module status in combined profile

### PR DESCRIPTION
**Pull request by jirihnidek**
* BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2058662
* Card ID: ENT-4778
* When modulemd is not in enabled or disabled, then the default
  status should be "default" not "unknown"
* Added file for unit tests of rhsm.profile and added several
  unit tests

### CODE REVIEW

File: src/rhsm/profile.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def __generate(self):
             all_module_list = modules.getModulePackages()
 
             for module_pkg in all_module_list:
-                status = "unknown"
+                status = "default"
                 active = False
```
Comment (Tomboyo) [inline src/rhsm/profile.py:153]: The BZ may not be clear about this, but I believe the active flag is incorrect as well. A module may be active even if it hasn't been explicitly enabled. On line 157, we're only setting the active flag if the module is also enabled, which means we'll treat some modules as inactive even though they're active and in the default state.

Comment (jirihnidek) [reply src/rhsm/profile.py:153]: Hi,
Thanks for feedback. It is written in the BZ. I only overlooked that. It should be fixed ATM.

Comment (Tomboyo) [reply src/rhsm/profile.py:153]: Looks good to me. Ty!

**Comment by ptoscano** [comment]
@jirihnidek, can you please run `black` on the new file? We have not setup CI jobs that check for black-compliant format yet, so we need to do this manually for now.

**Comment by jirihnidek** [comment]
@ptoscano updated and force pushed.

**Pull request merged**